module/Product/config/module.config.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Product;
+
+use Zend\Router\Http\Literal;
+use Zend\Router\Http\Segment;
+use Zend\ServiceManager\Factory\InvokableFactory;
+
+return [
+    'router' => [
+      'routes' => [
+          'product-list' => [
+              'type'    => Segment::class,
+              'options' => [
+                  'route'    => '/product/list[/:id]',
+                  'defaults' => [
+                      'controller' => Controller\ProductController::class,
+                      'action'     => 'index',
+                  ],
+              ],
+          ],
+          'product-add' => [
+              'type'    => Segment::class,
+              'options' => [
+                  'route'    => '/product/add',
+                  'defaults' => [
+                      'controller' => Controller\ProductController::class,
+                      'action'     => 'add',
+                  ],
+              ],
+          ],
+          'product-edit' => [
+              'type'    => Segment::class,
+              'options' => [
+                  'route'    => '/product/edit[/:id]',
+                  'defaults' => [
+                      'controller' => Controller\ProductController::class,
+                      'action'     => 'edit',
+                  ],
+              ],
+          ],
+          'product-add-ajax' => [
+              'type'    => Segment::class,
+              'options' => [
+                  'route'    => '/product/post',
+                  'defaults' => [
+                      'controller' => Controller\ProductController::class,
+                      'action'     => 'addAjax',
+                  ],
+              ],
+          ],
+          'product-edit-ajax' => [
+              'type'    => Segment::class,
+              'options' => [
+                  'route'    => '/product/update',
+                  'defaults' => [
+                      'controller' => Controller\ProductController::class,
+                      'action'     => 'updateAjax',
+                  ],
+              ],
+          ],
+          'product-delete' => [
+              'type'    => Segment::class,
+              'options' => [
+                  'route'    => '/product/delete',
+                  'defaults' => [
+                      'controller' => Controller\ProductController::class,
+                      'action'     => 'deleteAjax',
+                  ],
+              ],
+          ],
+      ],
+    ],
+    'controllers' => [
+        'factories' => [
+            Controller\ProductController::class => Controller\ProductControllerFactory::class,
+        ],
+    ],
+    'view_manager' => [
```

Comment:
See review for `module.config.php` for `Category` module. We don't need to repeat keys that exist in the `Application` module as all modules configs are merged together.